### ISSUE TITLE
Set global Options in nuxt.config.js

### ISSUE THREAD

**Issue by holnburger** [richardeschloss/nuxt-highcharts#12]
Thanks for your awesome work! 

I'm currently struggling with setting global options and default options within the `nuxt.config.js`.
I was trying to find an example what objects are expected. I'm not able to set global defaults or chart defaults currently. I wanted to change the global language for example. With my approach that's currently not possible.

Do you maybe have an example and is it possible to also set global options - for example a theme. 

```
  // Highcharts options
  highcharts: {
    exporting: true,
    // currently not working TODO
    chartOptions: {
      setOptions: {
        colors: ['#fff', '#000', '#ED561B', '#DDDF00', '#24CBE5', '#64E572', '#FF9655', '#FFF263', '#6AF9C4'],
        global: {
          lang: {
            months: [
              "Januar", "Februar", "März", "April", "May",
              "Juni", "Juli", "August", "September", "Oktober",
              "November", "Dezember"
            ],
            weekdays: [
              "Sonntag", "Montag", "Dienstag", "Mittwoch", "Donnerstag", "Freitag", "Samstag"
            ],
            shortMonths: [
              "Jan", "Feb", "Mrz", "Apr", "Mai", "Jun", "Jul", "Aug", "Sep", "Okt", "Nov", "Dez"
            ]
        }
      }
      }

    }
  }, 
```

**Comment by richardeschloss** [comment]
Hi, thanks for bringing up the issue, and it should be working. Here's the part of the code where it tries consuming that option:

https://github.com/richardeschloss/nuxt-highcharts/blob/888143c7dc5b31f01ef76f359ec2fbc8ed1b6f2a/highcharts/components.js#L45

Can you please re-try with the default chartOptions I used in the following passing test and let me know how far you get?
https://github.com/richardeschloss/nuxt-highcharts/blob/888143c7dc5b31f01ef76f359ec2fbc8ed1b6f2a/test/components.spec.js#L6

```js
const dfltOption = {
  chartOptions: { // <-- try this first. 
    caption: {
      text: 'Some caption'
    },
    chart: {
      type: 'spline'
    },
    xAxis: [{
      title: {
        text: 'xAxis title'
      }
    }],
    yAxis: [{
      title: {
        text: 'Values'
      }
    }],
    title: {
      text: `Some title`
    },
    subtitle: {
      text: `Some subtitle`
    },
    series: [{
      name: 'My series',
      data: [0, 1, 2, 3, 4]
    }]
  }
}
```
This set should produce a very basic spline chart. If that works, then the issue is with the original chartOptions being provided (the plugin does not merge global options with local ones). However, if there are still issues, I can try to troubleshoot later tonight.

**Comment by richardeschloss** [comment]
I think this will be addressed in #14 . The "setOptions" would not be in chartOptions. They would be in the highcharts options that would get passed to highcharts API method "setOptions".
